core/src/com/unciv/ui/pickerscreens/PickerScreen.kt

```diff
@@ -6,48 +6,45 @@ import com.unciv.UncivGame
 import com.unciv.ui.utils.*
 import com.unciv.ui.utils.AutoScrollPane as ScrollPane
 
-open class PickerScreen(val disableScroll: Boolean = false) : CameraStageBaseScreen() {
```

Comment:
That was there for a reason

Reply:
Tested and yes, this breaks at least load game and mod manager screens.
I admit there's other possible solutions - maybe those PickerScreen's actually using that ScrollPane are in the minority so the creation could be removed and refactored to those specifically - or another layer of subclass or or or...